packages/cadl-python/src/emitter.ts

```diff
@@ -326,6 +326,10 @@ function emitBodyParameter(
         type = getType(program, bodyType);
     }
 
+    if (type.type === "model" && type.name === "") {
+        type.name = capitalize(operation.name) + "Request";
+    }
+
```

Comment:
We need assign a name even though we don't generate the model in output  so that we could avoid error in `autorest.python`. (Without the change, emitter will meet error for https://github.com/Azure/azure-rest-api-specs/tree/feature/cognitiveservices/openai-cadl/specification/cognitiveservices/OpenAI.Inference